include-what-you-use.1

```diff
@@ -72,10 +72,33 @@ Maximum line length for includes.
 Note that this only affects the comments and their alignment, the maximum line
 length can still be exceeded with long filenames (default: 80).
 .TP
+.BI \-\-comments [=Verbosity]
```

Comment:
We have optargs spelled all-lower for `--prefix_header_includes`, so I think this could just be `verbosity`. Here and below